tools/sdk-readme-table-generator/TableGenerator/Reference.fs

```diff
@@ -107,20 +150,38 @@ let winMuslReferenceTemplate: ReferenceTemplate = {
 [{0}-zip-{1}]: https://dotnetcli.blob.core.windows.net/dotnet/Sdk/{2}/dotnet-sdk-latest-{0}.zip
 [{0}-zip-checksum-{1}]: https://dotnetclichecksums.blob.core.windows.net/dotnet/Sdk/{2}/dotnet-sdk-latest-{0}.zip.sha"""
 
-    AkaMSTemplate = """[{0}-badge-{1}]: https://aka.ms/dotnet/{4}/Sdk/{3}_Release_version_badge.svg
+    AkaMSLegacyTemplate = """[{0}-badge-{1}]: https://aka.ms/dotnet/{4}/Sdk/{3}_Release_version_badge.svg
 [{0}-version-{1}]: https://aka.ms/dotnet/{4}/Sdk/productCommit-{0}.txt
 [{0}-zip-{1}]: https://aka.ms/dotnet/{4}/Sdk/dotnet-sdk-{0}.zip
 [{0}-zip-checksum-{1}]: https://aka.ms/dotnet/{4}/Sdk/dotnet-sdk-{0}.zip.sha"""
+
+    AkaMSTemplate = """[{0}-badge-{1}]: https://aka.ms/dotnet/{4}/{3}_Release_version_badge.svg
+[{0}-version-{1}]: https://aka.ms/dotnet/{4}/productCommit-{0}.txt
+[{0}-zip-{1}]: https://aka.ms/dotnet/{4}/dotnet-sdk-{0}.zip
+[{0}-zip-checksum-{1}]: https://aka.ms/dotnet/{4}/dotnet-sdk-{0}.zip.sha"""
 }
 
 let formatTemplate (platform: String) (template: ReferenceTemplate) (branch: Branch): Option<string> =
     if branch.AkaMsChannel <> None then
-        Some (String.Format(template.AkaMSTemplate, 
-                            platform,                       // 0 - win-x64
-                            (branchNameShorten branch),     // 1 - 5.0.1xx-preview2
-                            branch.GitBranchName,           // 2 - 5.0.100-preview.2.20169.1
-                            (platform.Replace('-', '_')),   // 3 - win_64
-                            branch.AkaMsChannel.Value))     // 4 - 5.0/preview2
+        let useLegacyAkaMsTemplate = 
+            match getMajorMinor branch with
+            | Master -> false
+            | MajorMinor { Major = major; Minor = _minor; Patch = patch } when major = 5 && patch < 300 -> true
```

Comment:
although that only applies to 5.x.xxx, not 3.x.xxx?

Reply:
3.x.xxx are already on the legacy-legacy links, since they don't use the AkaMsChannel